WikiWatershed/model-my-watershed
# Three Equal Stages: Move Model alongside Analyze, Monitor

## Overview

Moves "Model" button and action to be another tab, like "Analyze" and "Monitor", to bring it on equal footing with the other two. The routing is not changed, and the behavior of the buttons is identical to before.

Connects #2978 

### Demo

![screen shot 2018-10-01 at 1 15 48 pm](https://user-images.githubusercontent.com/1430060/46304374-81da8b00-c57c-11e8-831c-7a9537ad12d8.png)

![2018-10-01 13 16 01](https://user-images.githubusercontent.com/1430060/46304386-8606a880-c57c-11e8-89a2-9444ade74b49.gif)

### Notes

A larger workflow revision is suggested in #2983, which may be attempted at a later date.

## Testing Instructions

* Check out this branch and `bundle`
* Open the site, select shape and proceed to Analyze
* Ensure there is no Model button in the sidebar footer
* Ensure the "Change Area" button takes full width, and clicking it works as before
* Go back to Analyze stage and select the Model tab. Ensure the text matches what was there before.
* Test creating projects using both models. Ensure they work same as before.
* Re-open saved projects. Ensure they open correctly.
